conf/services/api-hub.yml

```diff
@@ -0,0 +1,69 @@
+serviceName: The API Hub
+serviceDescription: You can use this service to allow users to discover APIs, consume APIs and manage credentials with self-serve capabilities.
+serviceDomain: admin.tax.service.gov.uk
+serviceUrl: /api-hub
```

Comment:
I think this needs changing to 'api-hub-frontend'. I ran up api-hub-frontend and accessibility-statement locally and clicked on the link and it didn't work until I changed the URL.

Reply:
It needs to be api-hub. The URL for the service in production is admin.tax.service.gov.uk/api-hub. See the documentation in the README...
serviceUrl: /icecreams                  # The relative URL to the service (omitting www.tax.service.gov.uk)

I think the configuration that might be wrong is this bit in api-hub-frontend's application.conf...

accessibility-statement{
  service-path = "/api-hub-frontend"
}

That should probably change to be "api-hub". That would then match the file name we've created here. The convention in this repo is to use the service name not the frontend name.

Reply:
Ah I see. Are you going to put a PR for the frontend now so it works when merged?

Reply:
Yup, I'll do that today